kmm/header/header.py

```diff
@@ -12,14 +13,16 @@ class Header(kmm.FunctionalBase):
 
     @staticmethod
     @validate_arguments
-    def from_path(path: Path):
+    def from_path(path: Path, raise_on_malformed_data: bool = True):
```

Comment:
`patch_malformed_data`?

Reply:
I liked having the same argument name everywhere, it looks a little strange passing the variable `raise_on_malformed_data` to the argument name `patch_malformed_data`

Reply:
Sure. My thinking was that we will be raising on malformed data :) This is an attempt to patch that will fail if it's something else that's wrong